### ISSUE TITLE
Quicklisp daily mirror

### ISSUE THREAD

**Issue by phoe** [ultralisp/ultralisp-projects#1]
Would it be a good idea to set up an Ultralisp dist that mimics the Quicklisp project list, except it checks for changes and rebuilds the dist every day?

The list of all projects in Quicklisp is available at https://github.com/quicklisp/quicklisp-projects/tree/master/projects - you could clone that every day and parse the entry for each directory.

**Comment by svetlyak40wt** [comment]
Good idea, I thought about something like that. However I need to add a database and support for datasources other than GitHub before.

Definitely, autoimport of projects from the official Quicklisp distribution is a "must have".
